demokratie-live/democracy-client
# sprint#16/performance

**Issue:** #362 

**Description:**
https://github.com/demokratie-live/democracy-client/issues/362

**How to test:**
Check if List changed after user events (voting, updateList, increase activity index, ...)


https://github.com/demokratie-live/democracy-development/pull/124

**Checklist:**
